src/app/core/cases/cases.service.ts

```diff
@@ -168,6 +169,26 @@ export class CasesService {
       });
   }
 
+  getCaseHistoryView(jurisdictionId: string,
```

Comment:
CasesService is in my opinion growing too much with too many responsibility. 
Can we introduce a CaseHistoryService and move this method there?

Reply:
Done